WidgetBus/Models/Network/BusLocationsOnRoute.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  BusLocationsOnRoute.swift
+//  WidgetBus
+//
+//  Created by Jaehwa Noh on 2022/11/17.
+//
+
+import Foundation
+
+struct BusLocationsOnRoute: Codable {
+    let response: BusLocationsResponse
+}
+
+struct BusLocationsResponse: Codable {
+    let header: BusLocationsResponseHeader
+    let body: BusLocationsResponseBody
+}
+
+struct BusLocationsResponseHeader: Codable {
+    let resultCode: String
+    let resultMsg: String
+}
+
+struct BusLocationsResponseBody: Codable {
+    let items: BusLocationsResponseItems
+    let numOfRows: Int
+    let pageNo: Int
+    let totalCount: Int
+}
+
+struct BusLocationsResponseItems: Codable {
+    let item: [BusLocationsInfo]
+}
+
+struct BusLocationsInfo: Codable {
+    // WGS84 위도 좌표
```

Comment:
```suggestion
    /// WGS84 위도 좌표
```

/가 3개면 자동완성에서 주석이 뜹니다 ~ ✨